doc/Installation/Installation-Ubuntu-1804-Nginx.md

```diff
@@ -85,75 +84,93 @@ systemctl restart mysql
 
 # Web Server
 
-## Configure and Start PHP-FPM
+## Configure and Restart PHP-FPM
 
-Ensure date.timezone is set in php.ini to your preferred time zone.
-See <http://php.net/manual/en/timezones.php> for a list of supported
-timezones.  Valid examples are: "America/New_York",
-"Australia/Brisbane", "Etc/UTC".
-Please remember to set the system timezone as well.
+Ensure `date.timezone` is set in `php.ini` to your preferred time zone.
+
+Valid examples are: `"America/New_York"`, `"Australia/Brisbane"`, `"Europe/Bratislava"`, `"Etc/UTC"`.
+
+>See <http://php.net/manual/en/timezones.php> for a list of supported timezones and change `Europe/Bratislava` in sed command to your timezone.
 
 ```bash
-vi /etc/php/7.2/fpm/php.ini
-vi /etc/php/7.2/cli/php.ini
+echo date.timezone = \"Europe/Bratislava\" >> /etc/php/7.4/fpm/php.ini
+echo date.timezone = \"Europe/Bratislava\" >> /etc/php/7.4/cli/php.ini
 ```
 
-```bash
-systemctl restart php7.2-fpm
-´´´
+Restart php-fpm
 
+```bash
+systemctl restart php7.4-fpm
 ```
-timedatectl set-timezone Etc/UTC
-´´´
 
 ## Configure NGINX
 
 ```bash
-vi /etc/nginx/conf.d/librenms.conf
+nano /etc/nginx/conf.d/librenms.conf
 ```
 
 Add the following config, edit `server_name` as required:
 
 ```nginx
 server {
- listen      80;
- server_name librenms.example.com;
- root        /opt/librenms/html;
- index       index.php;
-
- charset utf-8;
- gzip on;
- gzip_types text/css application/javascript text/javascript application/x-javascript image/svg+xml text/plain text/xsd text/xsl text/xml image/x-icon;
- location / {
-  try_files $uri $uri/ /index.php?$query_string;
- }
- location /api/v0 {
-  try_files $uri $uri/ /api_v0.php?$query_string;
- }
- location ~ \.php {
-  include fastcgi.conf;
-  fastcgi_split_path_info ^(.+\.php)(/.+)$;
-  fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
- }
- location ~ /\.ht {
-  deny all;
- }
+    listen      80;
+    server_name librenms.example.com;
+    root        /opt/librenms/html;
+    index       index.php;
+
+    charset utf-8;
+    gzip on;
+    gzip_types text/css application/javascript text/javascript application/x-javascript image/svg+xml text/plain text/xsd  text/xsl text/xml image/x-icon;
+ 
+    location / {
+        try_files $uri $uri/ /index.php?$query_string;
+    }
+ 
+    location /api/v0 {
+        try_files $uri $uri/ /api_v0.php?$query_string;
+    }
+ 
+    location ~ \.php {
+        include fastcgi.conf;
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+        fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
+    }
+ 
+    location ~ /\.ht {
+        deny all;
+    }
 }
 ```
 
+Delete default nginx site.
+
 ```bash
 rm /etc/nginx/sites-enabled/default
 systemctl restart nginx
 ```
 
+Check nginx configuration and restart nginx service.
+
+```bash
+nginx -t
```

Comment:
shouldn't this not be done after creating config for librenms and  before restarting nginx ?

Reply:
fixed